### ISSUE TITLE
Yifisubtitles unusual amount of requests / connectionerror all the time

### ISSUE THREAD

**Issue by moostmartijn** [morpheus65535/bazarr#2074]
**Describe the bug**
I setup yifisubtitles as a provider and did a manual search for a subtitle. It was not able to download the subtitle because of Max retries. Now it is stuck on trying to connect to yifisubtitles. There is a ConnectionError, but it's still trying to connect every 5s. The log is full of connection errors for yifisubtitles:

```
27/02/2023 09:52:48|INFO    |root                            |Throttling yifysubtitles for 10 minutes, until 23/02/27 10:02, because of: ConnectionError. Exception info: MaxRetryError("HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474537c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))")|
27/02/2023 09:52:53|INFO    |root                            |Throttling yifysubtitles for 10 minutes, until 23/02/27 10:02, because of: ConnectionError. Exception info: MaxRetryError("HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474536380>: Failed to establish a new connection: [Errno -2] Name or service not known'))")|
27/02/2023 09:52:54|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14745367a0>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:52:59|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14745355d0>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:52:59|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474534340>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:53:04|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474537640>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:53:04|INFO    |root                            |Throttling yifysubtitles for 10 minutes, until 23/02/27 10:03, because of: ConnectionError. Exception info: MaxRetryError("HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14745358d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))")|
27/02/2023 09:53:09|INFO    |root                            |Throttling yifysubtitles for 10 minutes, until 23/02/27 10:03, because of: ConnectionError. Exception info: MaxRetryError("HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14745364d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))")|
27/02/2023 09:53:10|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14745347c0>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:53:15|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474535720>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:53:15|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474537280>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:53:20|WARNING |retry.api                       |HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f14745372e0>: Failed to establish a new connection: [Errno -2] Name or service not known')), retrying in 5 seconds...|
27/02/2023 09:53:20|INFO    |root                            |Throttling yifysubtitles for 10 minutes, until 23/02/27 10:03, because of: ConnectionError. Exception info: MaxRetryError("HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474537fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))")|
27/02/2023 09:53:25|INFO    |root                            |Throttling yifysubtitles for 10 minutes, until 23/02/27 10:03, because of: ConnectionError. Exception info: MaxRetryError("HTTPSConnectionPool(host='yifysubtitles.orghttps', port=443): Max retries exceeded with url: //yifysubtitles.ch/subtitles/three-billboards-outside-ebbing-missouri-2017-dutch-yify-431579 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1474535720>: Failed to establish a new connection: [Errno -2] Name or service not known'))")|

```

This is just a small part of the log... There are 1000's more lines of errors in my log. In Adguard Home I noticed an unusual amount of requests to yifisubtitles in the last 24h:
<img width="561" alt="Schermafbeelding 2023-02-27 om 09 55 24" src="https://user-images.githubusercontent.com/31592734/221518054-93dbcd62-f48a-4fdd-8b80-44c0b122d754.png">

**To Reproduce**
1. Setup yifisubtitles
2. Manual search for subtitles and download
3. Wait for ConnectionError

**Expected behavior**
Solve the ConnectionError over time and download the subtitles.

**Screenshots**

**Software (please complete the following information):**
 - Bazarr: [e.g. v 0.6.1]: v1.1.2 (latest version not available as Synology package, manual update withing Bazarr not possible?)
 - Radarr version [e.g. v 0.2.0.0001]: Latest
 - Sonarr version [e.g. v 2.0.0.0001]: Latest
 - OS: Linux (Synology NAS)

**Additional context**

**Comment by ayancey** [comment]
There was a bug with the yify provider that was not fixed until v1.1.4. Are you using the SynoCommunity release? Ideally, you should run Bazarr with docker, which should be possible on your NAS. Thanks.

**Comment by halali** [comment]
update to latest version... we dont support outdated versions.
